### DISCUSSION TITLE
Errores de output en ejercicios del drive

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#68] [Tarea 1] [answered]
Buenas tardes, el los ejemplos del drive sólo se imprimen las transacciones y el estado del mercado, no se imprime nada al añadir un producto para vender/comprar, pero en el enunciado dice que sí hay que añadir al output cuando se registra una compra o venta 😅

**Comment by benjavicente** [comment, marked as answer]
Hola! Ahora deberían estar arreglados los test cases de referencia.

**Reply by MaxAl100** [reply]
Me parece que no están arreglados, ahora el SPREAD tiene problemas, no? 
Por ejemplo aquí:
Estado del mercado
	9 por 66
	SPREAD 1
	10 por 54
	
El SPREAD debiera ser 12, no?

**Reply by benjavicente** [reply]
Los de la izquierda son los montos.

**Reply by MaxAl100** [reply]
Ahhh ya, estaba entendiendo mal el output 😅 
Muchas gracias!
